Resources/config/services.xml

```diff
@@ -183,5 +185,10 @@
             <argument type="service" id="Trikoder\Bundle\OAuth2Bundle\Manager\ClientManagerInterface" />
         </service>
         <service id="trikoder.oauth2.event.authorization_request_resolve_event_factory" alias="Trikoder\Bundle\OAuth2Bundle\Event\AuthorizationRequestResolveEventFactory" />
+
+        <service id="Trikoder\Bundle\OAuth2Bundle\Security\Authentication\Token\OAuth2TokenFactory">
+            <argument type="string" />
+        </service>
+        <service id="trikoder.oauth2.security.authentication.token.oauth2_token_factory" alias="Trikoder\Bundle\OAuth2Bundle\Security\Authentication\Token\OAuth2TokenFactory" />
```

Comment:
@Allypost @spideyfusion Wouldn't add legacy aliases to new classes.

Reply:
@HypeMC Fixed